samples/bluetooth/st_ble_sensor/src/led_svc.c

```diff
@@ -26,24 +26,35 @@ LOG_MODULE_REGISTER(led_svc);
 struct device *led_dev;
 bool led_state;
 
-void led_on_off(u32_t led_state)
+void led_on_off(bool led_state)
 {
+	u32_t led_param = (led_state == true) ? 1 : 0;
```

Comment:
`gpio_pin_set` takes `int` as a parameter, `led_param` should be declared as `int`.